iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/models/DeploymentServiceModel.java

```diff
@@ -53,6 +61,14 @@ public DeploymentServiceModel(Configuration config) throws InvalidInputException
         this.id = config.getId();
         this.name = config.getLabels().get(DEPLOYMENT_NAME_LABEL);
         this.deviceGroupId = config.getLabels().get(DEPLOYMENT_GROUP_ID_LABEL);
+
+        if (config.getLabels().containsKey(DEPLOYMENT_GROUP_NAME_LABEL)) {
+            this.deviceGroupName = config.getLabels().get(DEPLOYMENT_GROUP_NAME_LABEL);
```

Comment:
Should this be initialized to empty string if it is not in config or is it ok if it is null?